app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -1300,13 +1305,26 @@ public void updateDisplay() {
             // Update the logo
             String imageUrl = ImageUtils.getLogoImageUrl(current, apiClient.getValue(), 440, false);
             if (imageUrl != null) {
-                Glide.with(requireContext())
-                        .load(imageUrl)
-                        .centerInside()
-                        .into(binding.itemLogo);
-                binding.itemLogo.setContentDescription(current.getName());
                 binding.itemLogo.setVisibility(View.VISIBLE);
                 binding.itemTitle.setVisibility(View.GONE);
```

Comment:
Shouldn't these by in the `onResourceReady` function?

Reply:
Placing them in the `onResourceReady` function causes the text to be pushed down while the logo is loading. Leaving them outside the function places the subtitle text where it should be once the logo is loaded so it doesn't shift before/after the logo is loaded. The subtitle text will still shift up a little if the logo fails to load since the title text has a smaller height than `itemLogo`, but ideally the logo will load successfully if we're in this section since the imageUrl is not null.

![Screenshot_1625393489](https://user-images.githubusercontent.com/46704654/124381700-e842e980-dc91-11eb-9be1-d9f44d8a9e65.png)